<launch>
	<node
		pkg="dual_quaternion_registration"
		type="qf_registration_server"
		name="qf_registration_server"
		output="screen"
		launch-prefix="xterm -e gdb --args"
	/>

</launch>